CONTRIBUTING.md

```diff
@@ -13,4 +13,4 @@ If you use Visual Studio Code, the editor will automatically provide validation
 
 ## Publishing
 
-Once your pull request is approved and merged, the application index will be rebuilt and republished automatically.
+Once your pull request is approved and merged, the add-on index will be rebuilt and published automatically.
```

Comment:
Do we call it `add-on`, or `add-ons` while mentioning multiple add-ons?

Reply:
`add-ons`, but here, `add-on` is a noun modifier.